src/main/java/carpet/CarpetSettings.java

```diff
@@ -739,4 +744,61 @@ public static void changeSpawnSize(int size)
             category = {SURVIVAL, CLIENT}
     )
     public static boolean cleanLogs = false;
+
+    public static class StructureBlockLimitValidator extends Validator<Integer> {
+
+        @Override public Integer validate(ServerCommandSource source, ParsedRule<Integer> currentRule, Integer newValue, String string) {
+            return (newValue >= 46) ? newValue : null;
```

Comment:
Did I say 46? I definitely meant 48. Can't imagine why that would be lower than that

Reply:
For some reason I keep messing simple integer recently. That's a typo.